lib/registries/addon/overview/index/template.hbs

```diff
@@ -3,9 +3,9 @@
         {{t 'registries.overview.component_of'}}
         <OsfLink
             @route='resolve-guid'
-            @models={{array this.root.id}}
+            @models={{array this.registration.root.id}}
         >
-            {{this.root.title}}
+            {{this.registration.root.title}}
```

Comment:
Calling `this.registration.root.title` here is triggering an additional AJAX call to fetch the `root` relationship. You may verify this in the network tab of the browser.

Reply:
Ok, to my understanding it should not be an AJAX call but should be a property on the model that in the Ember store. This can be viewed with Ember Inspector and clicking the reigstration on the Data tab. The registration is bound on the controller but is not making any additional async/awaits. Otherwise, I could not be fully understanding the way the template is retrieving the data. Let's follow up on a call to make sure this is properly being understood.